sources/common-dylan/tests/specification.dylan

```diff
@@ -486,6 +486,77 @@ define interface-specification-suite threads-specification-suite ()
   test test-atomic-increment!;
 end threads-specification-suite;
 
+define suite common-dylan-classes-test-suite ()
+  test test-<format-string-condition>;
+  test test-<simple-condition>;
+  test test-<arithmetic-error>;
+  test test-<division-by-zero-error>;
+  test test-<arithmetic-domain-error>;
+  test test-<arithmetic-overflow-error>;
+  test test-<arithmetic-underflow-error>;
+  test test-<stretchy-sequence>;
+  test test-<stretchy-object-vector>;
+  test test-<object-deque>;
+  test test-<string-table>;
+  test test-<random>;
+end;
+
+define suite common-dylan-functions-test-suite ()
+  test test-concatenate!;
+  test test-condition-to-string;
+  test test-debug-message;
+  test test-difference;
+  test test-false-or;
+  test test-find-element;
+  test test-float-to-string;
+  test test-ignorable;
+  test test-ignore;
+  test test-integer-to-string;
+  test test-number-to-string;
+  test test-one-of;
+  test test-position;
+  test test-split;
+  test test-join;
+  test test-remove-all-keys!;
+  test test-string-to-integer;
+  test test-subclass;
+  test test-fill-table!;
+  test test-application-name;
+  test test-application-filename;
+  test test-application-arguments;
+  test test-tokenize-command-line;
+  test test-exit-application;
+  test test-register-application-exit-function;
+  test test-unfound;
+  test test-unfound?;
+  test test-found?;
+  test test-unsupplied;
+  test test-unsupplied?;
+  test test-supplied?;
+  test test-true?;
+  test test-false?;
+  test test-format-out;
+  test test-format-to-string;
+  test test-random;
+  test test-start-profiling;
+  test test-start-profiling-type;
+  test test-stop-profiling;
+  test test-stop-profiling-type;
+  test test-profiling-type-result;
+  test test-drain-finalization-queue;
+  test test-finalize;
+  test test-finalize-when-unreachable;
+  test test-automatic-finalization-enabled?-setter;
+  test test-automatic-finalization-enabled?;
+  test test-integer-length;
+  test test-float-radix;
+  test test-float-digits;
+  test test-float-precision;
+  test test-decode-float;
+  test test-scale-float;
+  test test-classify-float;
+end;
+
```

Comment:
I probably should have reorganized these into a numerics suite, random suite, etc, but my plan is to delete the suites so I would rather just leave as is.

Reply:
I frequently use the existing suites and would prefer that they not be deleted.

Reply:
How do you use them in practice?  It seems unlikely that you would want to, say, run the tests for all the functions or run the tests for all the macros, and that's the current organization. My usual M.O. is to run the whole thing once and then run a single failing test until I make it pass.

At any rate, I'll re-org the suites instead of deleting them for now. I'll continue to try and come up with a better design than suites because I still believe https://gitter.im/dylan-lang/general?at=5cb4d72d016a930a45c58c8e. I think I have something reasonable in mind.

Reply:
Maybe I can't think of anything better after all. Best I can come up with is `define [suite-name] test test-name ()`.